app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -19,10 +23,15 @@ def destroy
     render json: {}, status: :no_content
   end
 
-  private 
+  private
   def authorize_user
     if !current_user.admin?
       raise ActionController::RoutingError.new("not found")
     end
   end
-end
\ No newline at end of file
+
+  def serialized_reviews_for_restaurant
```

Comment:
instead of serializing the reviews in this restaurants controller, I would recommend serializing the restaurant making sure to include the association with reviews

Reply:
just updated added in a serializer for the restaurant show API, with the association to reviews. still using a custom method to call on the serialized restaurant since we have a custom json with the user data